requirements_all.txt

```diff
@@ -171,6 +171,7 @@ freesms==0.1.1
 
 # homeassistant.components.device_tracker.fritz
 # homeassistant.components.sensor.fritzbox_callmonitor
+# homeassistant.components.sensor.fritzbox_callforwarding
```

Comment:
Actually I tried running the script before. It made a lot of changes, decremented a lot of deps etc. 
 # homeassistant.components.device_tracker.fritz
-# homeassistant.components.sensor.fritzbox_callmonitor
-# homeassistant.components.sensor.fritzbox_callforwarding
 # homeassistant.components.sensor.fritzbox_netmonitor
-# fritzconnection==0.6.3
+# fritzconnection==0.6

It seems to have picked up and evaluated an old version of hass so I attempted to alter the file manually.